src/Writer/XLSX/Options.php

```diff
@@ -63,4 +71,65 @@ public function getMergeCells(): array
     {
         return $this->MERGE_CELLS;
     }
+
+    /**
+     * set Worksheets page margins.
+     *
+     * @param float $top    inches
+     * @param float $right  inches
+     * @param float $bottom inches
+     * @param float $left   inches
+     * @param float $header inches
+     * @param float $footer inches
+     */
+    public function setPageMargin(
+        float $top = 0.75,
+        float $right = 0.7,
+        float $bottom = 0.75,
+        float $left = 0.7,
+        float $header = 0.3,
+        float $footer = 0.3
+    ): void {
+        $this->pageMargin = new PageMargin(
+            $top,
+            $right,
+            $bottom,
+            $left,
+            $header,
+            $footer,
+        );
+    }
+
+    public function getPageMargin(): PageMargin
+    {
+        return $this->pageMargin;
+    }
+
+    public function hasPageMargin(): bool
+    {
+        return (bool) $this->pageMargin;
+    }
+
+    public function setPageOrientation(EnumPageOrientation $orientation): void
```

Comment:
Same as `PageMargin` we can introduce a `PageSetup` class that accepts both orientation and papere size, and a unique `setPageSetup` method, so we can get rid of the array.